app/Models/Contest.php

```diff
@@ -54,4 +54,14 @@ public function getCSVExport()
 
         return buildCSV($data);
     }
+
+    /**
+     * Set the sender attribute for the contest.
+     *
+     * @param string $value
+     */
+    public function setSenderAttribute($value)
+    {
+        $this->attributes['sender'] = ! empty($value) ? $value : null;
```

Comment:
This is just a mutator so if the field isn't filled out it sets the item in the DB to `null` instead of an empty string.